app/main/posts/views/filters/filters-dropdown.directive.js

```diff
@@ -16,20 +16,30 @@ function FiltersDropdown(PostFilters, ModalService, $rootScope) {
     };
 
     function FiltersDropdownLink($scope, $element, $attrs, ngModel) {
-        PostFilters.reactiveFilters = 'disabled';
+        PostFilters.reactiveFilters = false;
         // Init an empty saved search
         $scope.savedSearch = {
             view : 'map',
             role : []
         };
+
+        // Check if we can edit
+        $scope.canEdit = function (savedSearch) {
+            return _.contains(savedSearch.allowed_privileges, 'update');
+        };
         $scope.applyFiltersLocked = function () {
-            PostFilters.reactiveFilters = 'enabled';
+            PostFilters.reactiveFilters = true;
```

Comment:
this is just some sane refactoring along the way because for some reason I had written enabled instead of actually using a boolean 👎